analysers/analyser_merge_parking_FR_BNLS.py

```diff
@@ -0,0 +1,73 @@
+
+
+class Analyser_Merge_Parking_FR_BNLS(Analyser_Merge):
+    def __init__(self, config, logger = None):
+        Analyser_Merge.__init__(self, config, logger)
+        self.missing_official = self.def_class(item = 8130, id = 1, level = 3, tags = ['merge', 'parking'],
+            title = T_f('{0} parking not integrated', 'BNLS'),
+            detail = T_(
+'''This parking is referenced in the database of car parks managed by local authorities in France, off-street.
+This database is made up of local authorities and is available on data.gouv.fr.'''),
+            fix = T_(
+'''See [the mapping](https://wiki.openstreetmap.org/wiki/France/data.gouv.fr/Base_nationale_des_lieux_de_stationnement)
+on the  wiki. Add a node or add tags if already existing.'''))
+        self.possible_merge = self.def_class(item = 8131, id = 3, level = 3, tags = ['merge', 'parking'],
+            title = T_('{0} parking integration suggestion'), 'BNLS')
+        self.update_official = self.def_class(item = 8132, id = 4, level = 3, tags = ['merge', 'parking'],
+            title = T_('{0} parking  update'), 'BNLS')
+
+        self.init(
+            "https://www.data.gouv.fr/fr/datasets/base-nationale-des-lieux-de-stationnement/",
+            "Base Nationale des Lieux de Stationnement",
+            CSV(Source(attribution = "Équipe du Point d'Accès National", millesime = "2020/04/30",
+                    fileUrl = "https://www.data.gouv.fr/fr/datasets/r/9723bb08-b38d-4ce6-88a9-81f3e366d316"),
+                separator = ";"),
+            Load("Xlong", "Ylat",
+                xFunction = self.float_comma,
+                yFunction = self.float_comma),
+            Mapping(
+                select = Select(
+                    types = ["nodes", "ways"],
+                    tags = {"amenity": "parking"}),
+                conflationDistance = 200,
+		  osmRef = "ref:FR:BNLS",
+                generate = Generate(
+                    static1 = {
+                        "amenity": "parking"},
+                    static2 = {
+                        "source": self.source},
+                    mapping1 = {
+                        "ref:FR:BNLS": "id",
+                        "name": "nom",
+                        "website": "url",
+                        "access": lambda res: "yes" if res["type_usagers"] == "tous" else "customers" if res["type_usagers"] == "abonnés" else None,
+                        "fee": lambda res: "yes" if res["gratuit"] == "0" else "no",
+                        "capacity": lambda res: res["nb_places"] if res["nb_places"] != "0" else None,
+			  "capacity:disabled": lambda res: res["nb_pmr"] if res["nb_pmr"] != "0" else None,
+			  "capacity:charging": lambda res: res["nb_voitures_electriques"] if res["nb_voitures_electriques"] != "0" else None,
+                        "maxheight": lambda res: int(res["hauteur_max"]) / 10 if res["hauteur_max"] != "N/A" else None,
```

Comment:
Hauteur maximale autorisée à la fois pour l'accès au parking et pour le stationnement du véhicule, en **centimètres**
```suggestion
                        "maxheight": lambda res: int(res["hauteur_max"]) / 100 if res["hauteur_max"] != "N/A" else None,
```

Reply:
merci !